rdrf/rdrf/templates/rdrf_cdes/patients_listing.html

```diff
@@ -127,7 +127,7 @@
                     columns: {{columns|safe }}
```

Comment:
The column heading icons seem to be using a glyphicon font that has since been removed, so they aren't being rendered correctly.

Reply:
It is rendering for me. Maybe try a hard refresh?
![image](https://user-images.githubusercontent.com/60163872/133340296-8fc952bf-c00f-4116-8dbb-b65091bd7429.png)

Reply:
Yep, my bad!